Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -176,7 +176,8 @@ def bump_formula_pr
       rsrc.version = forced_version if forced_version
       odie "No version specified!" unless rsrc.version
       rsrc_path = rsrc.fetch
-      if Utils.popen_read("/usr/bin/tar", "-tf", rsrc_path) =~ %r{/.*\.}
+      tar = which("gtar") || which("tar")
+      if Utils.popen_read(tar, "-tf", rsrc_path) =~ %r{/.*\.}
```

Comment:
How about something like:

```
gnu_tar_gtar_path = HOMEBREW_PREFIX/"opt/gnu-tar/bin/gtar"
gnu_tar_gtar = gnu_tar_gtar_path if gnu_tar_gtar_path.executable?
tar = which("gtar") || gnu_tar_gtar || which("tar")
```

This will let it use our `gtar` on systems where it doesn't exist and there's no system version.